pkgs/development/tools/rust/ra-multiplex/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+}:
+rustPlatform.buildRustPackage {
+  pname = "ra-multiplex";
+  version = "0.2.2";
+  cargoSha256 = "sha256-Z0ul50LeSb3qhdfuvRYvNAExtfH8y2+IkKLtqtlaL9E=";
+
+  src = fetchFromGitHub {
+    owner = "pr2502";
+    repo = "ra-multiplex";
+    rev = "dcb5f83890cb91016b0a1590cc1b732606bb6ec1";
+    sha256 = "sha256-Hf4Gj9eXEP4gXiqNV4Jq0oiGLX3DtDF9At1feEZ+bUE=";
+  };
+
+  # We make two extra scripts:
```

Comment:
Where are these scripts from? They do not seem to be required, and are not mentioned upstream. If they are just to make *YOU* easier to use, they should go to your personal profile.